DataFormats/GEMRecHit/src/ME0Segment.cc

```diff
@@ -5,6 +5,9 @@
  */
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "DataFormats/GEMRecHit/interface/ME0Segment.h"
+#include "Geometry/GEMGeometry/interface/ME0Chamber.h"
+#include "Geometry/GEMGeometry/interface/ME0Layer.h"
+#include <TVector2.h>
```

Comment:
@nickmccoll - isn't there a CMS equivalent for it in DataFormats/GeometryVector?